src/components/shared/MintButton.tsx

```diff
@@ -74,15 +75,21 @@ const MintButtonComponent: React.StatelessComponent<Props> = props => {
       />
       <Text style={{ marginTop: 4 }}>
         <MixedText
-          style={[fontSizes.small]}
+          style={[fontSizes.small, { textAlign: "center" }]}
           content={[
-            "The current mint rate is",
+            "Create ",
```

Comment:
Do we want to use the word "Create" or "Mint"? We keep switching between the two, i think we should choose one and commit. Mint is more conceptually compelling but Create is clearer.